os/arch/arm/src/armv7-r/arm_fullcontextrestore.S

```diff
@@ -134,6 +134,12 @@ up_fullcontextrestore:
 	 * to either user or kernel mode.
 	 */
 
+	sub	sp, #4
```

Comment:
I think this code must be under PROTECTED config or otherwise removed.  Please check and modify.

Reply:
Already under PROTECTED flag.